lensless/recon.py

```diff
@@ -114,6 +114,26 @@ class :py:class:`~lensless.ReconstructionAlgorithm`. The three reconstruction
     python scripts/recon/admm.py -cn <CONFIG_FILENAME_WITHOUT_YAML_EXT>
 
 
+3D example
+----------
+
+It is also possible to reconstruct 3D scenes using Gradient Descent or APGD. ADMM doesn't supports 3D reconstruction yet.
```

Comment:
render algorithms with hyperlinks as
```
:py:class:`~lensless.GradientDescent`
:py:class:`~lensless.ADMM`
:py:class:`~lensless.APGD`
```
as in [here](https://github.com/LCAV/LenslessPiCam/blob/4a84625fa2d4081c34294a9c34b3b3af835fb47c/lensless/recon.py#L16)